src/App.vue

```diff
@@ -109,18 +118,28 @@ export default {
   },
   methods: {
     ...mapActions('elementalChat', ['setChannelPolling', 'updateProfile']),
-    ...mapActions('holochain', ['skipBackoff']),
+    ...mapActions('holochain', ['skipBackoff', 'holoLogout']),
     ...mapMutations(['setAgentHandle', 'setErrorMessage']),
+    ...mapMutations('elementalChat', ['setNeedsHandle']),
     agentHandleEntered () {
       if (this.internalAgentHandle === '') return
       this.updateProfile(this.internalAgentHandle)
       this.dialog = false
     },
+    closeModal () {
+      this.setNeedsHandle(false)
+    },
```

Comment:
How can we allow users to close the nickname modal without having a nickname? Then we'll get messages posted with `null` nicknames or whatever.

Reply:
ha, yes... I thought about this too and already have a solution in mind.  I'll push up a small update for this.